pkg/queue/queue_test.go

```
@@ -32,7 +32,7 @@ const (
 )
 
 func TestFIFOClusterQueue(t *testing.T) {
-	q := newClusterQueue(&kueue.ClusterQueue{
```

Don't discard the error.
Instead, call t.Fatal if one shows up. Same in other locations.

---

Done.
Thanks.